lms/static/styles/components/_Dialog.scss

```diff
@@ -1,32 +1,22 @@
-.Dialog__container {
+.Dialog__background {
   display: flex;
-  flex-direction: column;
   justify-content: center;
```

Comment:
According to [this guide](https://css-tricks.com/centering-css-complete-guide/) you need both justify-content and align-items set to center to center vertically and horizontally. 
![image](https://user-images.githubusercontent.com/30059933/63615949-5baa4900-c59b-11e9-9897-c92e2ed044a9.png)
I'm also finding this to be true just experimenting with it locally.

Reply:
> I mentioned in my earlier comments that vertically centering the content may be a problem if the iframe ends up being so tall for some reason that the top of the dialog ends up off-screen. If that does come to pass, one way to resolve that would be to position the background's content at the top of the dialog but add a vertical spacer (using flex-basis and flex-shrink) that stretches up to say 200px (if the content is short) or shrinks if the content is tall.

So what you're getting at here is that it might be better to have the content within the Dialog be of a minimum a certain size and of a maximum a certain size such that you can see the whole of the content without needing to scroll (aka: title to button). I believe these would need to be set by the content containing div such as ErrorDialog and not the Dialog div though. If you set them on the Dialog content div the space is added at the bottom which ends up looking something like this:
![image](https://user-images.githubusercontent.com/30059933/63617497-82b64a00-c59e-11e9-858f-391925eb1c16.png)

Reply:
Note I have also added a little wrapper with the min-height set to 0 to prevent the top from overflowing upwards if the height of the content exceeds the ancestor's height as per https://css-tricks.com/flexbox-truncated-text/.

Reply:
> According to this guide you need both justify-content and align-items set to center vertically and horizontally.

Right, sorry. That's correct. Flexbox lays items out in either a vertical or horizontal line. `justify-content` then controls the alignment of an item in the direction that the line is laid out and `align-items` controls the alignment in the perpendicular direction.